cmd/vclusterctl/cmd/platform/import/import.go

```diff
@@ -0,0 +1,25 @@
+package vimport
```

Comment:
Why are we calling the package different than the folder name?

Reply:
I couldn't name the package import because import is a reserved keyword in Go... 🙈